src/screens/Home/Home.js

```diff
@@ -250,17 +238,8 @@ function Home() {
             testIdTag={TAG}
           />
 
-          {/* this should stay first element, avoid putting it inside UserNameAndImage */}
-          {canSwitchAccount && (
-            <Tooltip
-              isVisible={!accountSwitchTooltipDismissed && showAccountSwitchTooltip}
-              body={t('tooltip.switchAccountsByTappingHere')}
-              wrapperStyle={{ zIndex: 9999, top: -10, position: 'relative' }}
-            />
-          )}
-
           <Tooltip
-            isVisible={!switchAccountTooltipDismissed && isKeyBasedAccount && !showAccountSwitchTooltip}
+            isVisible={!switchAccountTooltipDismissed && isKeyBasedAccount}
             body={t('tooltip.switch_account')}
             wrapperStyle={{ zIndex: 9999, top: -10, position: 'relative' }}
           />
```

Comment:
Could this also be removed?

Reply:
This is not mentioned in ticket. I think this is part of wallet connect. toast message is "Switched to your Key Wallet for the Etherspot Dashboard"